sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ViewDescriptorFactorySpec.scala

```diff
@@ -374,19 +375,38 @@ class ViewDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSuit
     }
 
     "generate proto for a View with multiple methods to handle different events" in {
-      assertDescriptor[SubscribeOnTypeToEventSourcedEventsWithMethodWithState] { desc =>
-        val methodOptions = this.findKalixMethodOptions(desc, "KalixSyntheticMethodOnESEmployee")
+      assertDescriptor[SubscribeOnTypeToEventSourcedEvents] { desc =>
 
-        val entityType = methodOptions.getEventing.getIn.getEventSourcedEntity
-        entityType shouldBe "employee"
+        val serviceOptions = findKalixServiceOptions(desc)
+        val eveningIn = serviceOptions.getEventing.getIn
+        eveningIn.getEventSourcedEntity shouldBe "employee"
 
+        val methodOptions = this.findKalixMethodOptions(desc, "KalixSyntheticMethodOnESEmployee")
         methodOptions.getView.getUpdate.getTable shouldBe "employee_table"
         methodOptions.getView.getUpdate.getTransformUpdates shouldBe true
         methodOptions.getView.getJsonSchema.getOutput shouldBe "Employee"
         methodOptions.getEventing.getIn.getIgnore shouldBe false // we don't set the property so the proxy won't ignore. Ignore is only internal to the SDK
 
       }
     }
+
+    "generate mappings for service to service subscription " in {
+      assertDescriptor[EventStreamSubscriptionView] { desc =>
+
+        val serviceOptions = findKalixServiceOptions(desc)
+        val eventingInDirect = serviceOptions.getEventing.getIn.getDirect
```

Comment:
Maybe adding `serviceOptions.getEventing.getIn.getIgnore shouldBe false`? This maybe not necessary but could be quite useful if it turns out we have to avoid passing `ignore = true` to the proxy. As I pointed out above. If that is the case could be nice to set the Publish.Stream to ignore true and verify here that is still false.